oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/DocumentPropertyStateTest.java

```diff
@@ -60,6 +73,11 @@ public void before() throws Exception {
         ns = builderProvider.newBuilder().setBlobStore(bs).getNodeStore();
     }
 
+    @After
+    public void tearDown() {
+        ns.dispose();
```

Comment:
```suggestion
        try {
            ns.dispose();
        } finally {
            DocumentPropertyState.setCompressionThreshold(DISABLED_COMPRESSION);
        }
```

to avoid side-effects, the test should ensure it doesn't leak a non default compression threshold to other tests. DISABLED_COMPRESSION is for illustration to refer to -1 - ideally the -1 would only be defined in the source class...

Reply:
Done